docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_AttentionOCR_From_Tensorflow.md

```diff
@@ -1,26 +1,26 @@
-# Convert TensorFlow Attention OCR Model {#openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_AttentionOCR_From_Tensorflow}
+# Converting a TensorFlow Attention OCR Model {#openvino_docs_MO_DG_prepare_model_convert_model_tf_specific_Convert_AttentionOCR_From_Tensorflow}
 
-This tutorial explains how to convert the Attention OCR (AOCR) model from the [TensorFlow* Attention OCR repository](https://github.com/emedvedev/attention-ocr) to the Intermediate Representation (IR).
+This tutorial explains how to convert the Attention OCR (AOCR) model from the [TensorFlow Attention OCR repository](https://github.com/emedvedev/attention-ocr) to the Intermediate Representation (IR).
 
-## Extract Model from `aocr` Library
+## Extracting a Model from `aocr` Library
 
-The easiest way to get an AOCR model is to download `aocr` Python\* library:
+To get an AOCR model, download `aocr` Python library:
 ```
 pip install git+https://github.com/emedvedev/attention-ocr.git@master#egg=aocr
 ```
-This library contains a pretrained model and allows to train and run AOCR using the command line. After installing `aocr`, you can extract the model:
+This library contains a pretrained model and allows training and running AOCR, using the command line. After installation of `aocr`, extract the model:
 ```
 aocr export --format=frozengraph model/path/
 ```
-After this step you can find the model in model/path/ folder.
+Once extracted, the model can be found in `model/path/` folder.
 
-## Convert the TensorFlow* AOCR Model to IR
+## Converting the TensorFlow AOCR Model to IR
 
-The original AOCR model contains data preprocessing which consists of the following steps:
+The original AOCR model contains data preprocessing, done as follows:
 * Decoding input data to binary format where input data is an image represented as a string.
 * Resizing binary image to working resolution.
 
-After that, the resized image is sent to the convolution neural network (CNN). The Model Optimizer does not support image decoding so you should cut of preprocessing part of the model using '--input' command line parameter.
+After that, the resized image is sent to the convolution neural network (CNN). The Model Optimizer does not support image decoding. The preprocessing part of the model should be cut off by using `--input` command line parameter.
```

Comment:
```suggestion
The resized image is sent to the convolution neural network (CNN). Because Model Optimizer does not support image decoding, the preprocessing part of the model should be cut off, using the `--input` command-line parameter.
```